keyboards/flutterby/flutterby.h

```diff
@@ -0,0 +1,33 @@
+#ifndef FLUTTERBY_H
+#define FLUTTERBY_H
+
+#include "quantum.h"
+
+#define KEYMAP( \
```

Comment:
We've changed the terminology here.  Could you please update this to reflect that change? 

```suggestion
#define LAYOUT( \
```